healthworkforce/mock-services/README.md

```diff
@@ -0,0 +1,43 @@
+# mCSD endpoint flow
+
+This guide will assist in getting the mCSD endpoint setup and tested with the mock service to return the relevant lookup results
+
+## Create Endpoint record
+
+```sh
+curl -X POST -d "@endpoint.json" -H "Content-Type: application/json" http://localhost:3003/endpoints
```

Comment:
@bradsawadye After removing the duplicate endpoint.json file this reference needs to point to the remaining file.

Reply:
@bradsawadye